### PR TITLE
Update authors and mailmap

### PR DESCRIPTION
**Pull request by max8899**
**- What I did**
Update authors and mailmap

**- How I did it**
./hack/generate-author.sh

**- How to verify it**

### CODE REVIEW

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks! Looks like some updates are needed to the `.mailmap` file; after that file is update, run the generate script again.

I'll post the required changes below 👍

**Comment by max8899** [comment]
@thaJeztah thank you for the help! And the pr is updated as you suggested, the diff file really helps 👍

File: AUTHORS [hunk 1]
```diff
@@ -845,6 +860,7 @@ Jeremy Unruh <jeremybunruh@gmail.com>
 Jeremy Yallop <yallop@docker.com>
 Jeroen Franse <jeroenfranse@gmail.com>
 Jeroen Jacobs <github@jeroenj.be>
+Jess Frazelle <acidburn@microsoft.com>
```
[the remark this answers is not in the document]
Comment (jessfraz) [reply AUTHORS]: :)

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**